<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_information_of_customer</class>
 <widget class="QDialog" name="add_information_of_customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(70, 255, 172);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>393</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(77, 77, 77);</string>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(48, 62, 72);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(48, 62, 72);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(48, 62, 72);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(48, 62, 72);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 10pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Enter the name of Customer :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 10pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Enter thelast name of Customer :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 10pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Enter the phone number Customer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 10pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Enter name product for record:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(77, 77, 77);
font: 700 10pt &quot;Segoe UI&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 10pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Enter the date of order :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 10pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Enter the number product of record:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(48, 62, 72);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>add_information_of_customer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
